packages/mobile/src/import/saga.ts

```diff
@@ -32,41 +45,71 @@ import { assignAccountFromPrivateKey, waitWeb3LastBlock } from 'src/web3/saga'
 
 const TAG = 'import/saga'
 
+export const MAX_BALANCE_CHECK_TASKS = 5
+export const MNEMONIC_AUTOCORRECT_TIMEOUT = 5000 // ms
+
 export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackupPhraseAction) {
   Logger.debug(TAG + '@importBackupPhraseSaga', 'Importing backup phrase')
   yield call(waitWeb3LastBlock)
   try {
-    const mnemonic = formatNonAccentedCharacters(phrase)
-    if (!validateMnemonic(mnemonic, bip39)) {
+    const normalizedPhrase = normalizeMnemonic(phrase)
+    const phraseIsValid = validateMnemonic(normalizedPhrase, bip39)
```

Comment:
Not sure how `suggestMnemonicCorrections` works exactly but if the `normalizedPhrase` has != 24 words will it have any chance of succeeding? If not maybe we can skip the error correction attempt. This might also be interesting to log with an analytics event (number of words in input phrases) :)

Reply:
So because of a check earlier in the process, phrases with more or less than 24 words will not make it this far. https://github.com/celo-org/wallet/blob/e70445836c53879696a780dc9864f5b322f1aa77/packages/mobile/src/backup/utils.ts#L102-L108

However, the `suggestMenmonicCorrections` function should that into account. In theory it could attempt to fix x phrase with too many or few words, so I wouldn't want to forgo passing such a phrase to the correction code. I've opened the following PR to at least avoid wasting time trying to correct the phrase since there is not currently a rule for it.
https://github.com/celo-org/celo-monorepo/pull/8146